nuxeo-dev-apb/apb.yml

```diff
@@ -95,7 +95,12 @@ plans:
       - name: nuxeo_version
         default: '9.10'
         type: enum
-        enum: ['8.10', '9.10', '10.2', '10.3', '10.10', '11.3', '11.4', 'master']
+        enum: [
```

Comment:
I think 1 version of 2021 is sufficient in my opinion. Once a sandbox is deployed we can bump the version.

Reply:
Added only 2021.20 to `nuxeo_version` parameter values in `default` plan

Reply:
updated `nuxeo_version` parameter values (removed obsolete values) in `default` plan